core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -46,3 +46,5 @@ workflows:
     platform: aws
   openshift-upgrade-azure-single-node:
     platform: azure
+  ipi-aws-windows:
```

Comment:
@bradmwilliams,  this lunch-workflow configuration will provision and de-provision a cluster with a Windows Node in AWS. It's sort of a no-op installation. I'm leaning toward adding some e2e for Windows and wrapping this workflow in something like `openshift-e2e-aws-windows`. WDYT?

Lasty, should I add a new `windows-node` variant in cluster-bot in addition to this e2e lunch workflow?

- https://github.com/openshift/ci-chat-bot/pull/251

Reply:
@jrvaldes A couple things.  Rearranging a bit to hopefully clarify things a bit...

> should I add a new `windows-node` variant in cluster-bot 

The `launch-workflow` command is, ideally, a convenient way for folks, like yourself, to test out their respective workflows, without actually exposing it to everyone as a supported variant.  That being said, adding the same workflow as a new variant, to the cluster-bot, pretty much negates the need for the `launch-workflow` changes all together.  So, I'd say pick the venue that works best for you and run with it.  I will mention, since is was asked in the other ci-chat-bot PR:

>   So what happen for `launch 4.9.9 vsphere,windows-node`?

It's virtually impossible to support every variant on every platform.  So, in places where a specific variant may not be supported, the cluster-bot, defaults down to it's `legacy` launch jobs (i.e. original, template based jobs, that are not supported any more), and you'll get a vanilla cluster running the requested version. 

> I'm leaning toward adding some e2e for Windows and wrapping this workflow in something like openshift-e2e-aws-windows. WDYT?

As far as the cluster-bot is concerned, it's really up to you.  The cluster-bot will support it either way (as just a workflow or a workflow with tests, as seen by the 2 different launch variants here:
https://github.com/openshift/release/blob/a5aa4d9bd7610e2e1da017c7a5a54d91c75fe4ce/ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml#L253-L268

Hopefully, this helps!  If you have any questions, please let me know!

Reply:
Indeed, thanks for the feedback! 

> ... adding the same workflow as a new variant, to the cluster-bot, pretty much negates the need for the `launch-workflow` changes all together.


Agree, and wasn't planning to add the "same" workflow.



For clarification, I was thinking of adding a new `windows-node` variant to cluster-bot to leverage the `- ref: clusterbot-wait` test. So that, cluster-bot consumers are aware of Windows as a new supported variant (for example by invoking `/help` command), can spin up, and play with a cluster with a Windows node that will last as long as $CLUSTER_DURATION.

On the other hand, I can see `launch-workflow` for testing Windows-specific workflows, for example:  `launch-workflow openshift-e2e-aws-windows`, that won't be exposed to everyone.

Finally, I feel we can leverage both, a new `launch-workflow` and a new `variant` with different workflows.

Reply:
To execute test, you could make the necessary changes to run the `/test` command with your respective tests.

Reply:
That looks good to me!